src/Scribunto/ScribuntoLuaLibrary.php

```diff
@@ -33,6 +33,28 @@ public function register(): void {
 		$this->getEngine()->registerInterface( __DIR__ . '/' . 'mw.wsslots.lua', $interfaceFuncs, [] );
 	}
 
+	/**
+	 * @param \WikiPage $wikiPage
+	 *
+	 * @return bool
+	 */
+	private function userCan( \WikiPage $wikiPage ): bool {
+		// Only do a check for user rights when not in cli mode
+		if ( PHP_SAPI != 'cli' ) {
+			$userCan = MediaWikiServices::getInstance()->getPermissionManager()->userCan(
+				'read',
+				RequestContext::getMain()->getUser(),
+				$wikiPage->getTitle()
+			);
+
+			if ( !$userCan ) {
```

Comment:
Can be simplified to `return $userCan;` (when taking into account other comments).